Dockerfile.dev

```diff
@@ -17,6 +17,7 @@ RUN npm install
 ## install server dependencies
 WORKDIR /app/server
 RUN npm install
+RUN empirica yarn add node-fetch
```

Comment:
I've updated the yarn-lock and dockerfile in main so that we shouldn't need this. If you merge main into your branch, it should overwrite this dockerfile change and (hopefully!) work!